vendor/github.com/grantseltzer/golumns/columns.go

```diff
@@ -62,5 +63,9 @@ func numberOfRows(numberOfRows, numberOfNames int) int {
 }
 
 func reverse(input []int) []int {
```

Comment:
It certainly is a little sloppy to have a helper function do it, but I chose it for readability reasons. The `sort.Reverse` function requires an interface type, so I'd have to do conversions with `sort.IntSlice` which I wasn't very happy with. I'll take another look to see if it's less complicated than I originally thought it was.